src/Console/Command/CommandHelp.php

```diff
@@ -286,6 +286,8 @@ public static function getDisplayableAllowedValues(FixerOption $option)
                 return !is_callable($value);
             });
 
+            sort($allowed);
```

Comment:
when one want to set up some fixed order that it would makes sense - the order always would be problematic, one would say `none` should be last, one would say it should be first.
when it's alphabetic, it's simple.